### PR TITLE
unify the crds used by helm chart and the installation scripts

### PR DESCRIPTION
**Pull request by jrkeen**
Signed-off-by: jrkeen <jrkeen@hotmail.com>

**What type of PR is this?**
/kind cleanup

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by jrkeen** [comment]
cc @RainbowMango @algebra2k @lfbear

**Comment by RainbowMango** [comment]
So quick! Thanks. LGTM.

**Comment by RainbowMango** [comment]
/assign @algebra2k 
Would you like to take a look?

**Review by algebra2k — COMMENTED** [review]
> /assign @algebra2k
> Would you like to take a look?

okay, i would try to reviewer

**Comment by jrkeen** [comment]
Installation scripts:
```
root@DESKTOP-TQOTF9Q:/workspace/projects/karmada# hack/remote-up-karmada.sh /root/.kube/config huawei-dev1
Generating a RSA private key
....................+++++
...................+++++
writing new private key to '/root/.karmada/server-ca.key'
-----
2021/09/10 17:00:37 [INFO] generate received request
2021/09/10 17:00:37 [INFO] received CSR
2021/09/10 17:00:37 [INFO] generating key: rsa-2048
2021/09/10 17:00:37 [INFO] encoded CSR
2021/09/10 17:00:37 [INFO] signed certificate with serial number 443313073895049221857854521357943746403274820507
Warning: resource namespaces/karmada-system is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
namespace/karmada-system configured
serviceaccount/karmada-controller-manager created
serviceaccount/karmada-scheduler created
serviceaccount/karmada-webhook created
clusterrole.rbac.authorization.k8s.io/karmada-controller-manager created
clusterrolebinding.rbac.authorization.k8s.io/karmada-controller-manager created
secret/karmada-cert-secret created
secret/kubeconfig created
secret/webhook-cert created
statefulset.apps/etcd created
service/etcd-client created
service/etcd created
wait the etcd ready...
...
```
Helm:
```
root@DESKTOP-TQOTF9Q:/workspace/projects/karmada# helm install karmada -n karmada-system --create-namespace ./charts/
NAME: karmada
LAST DEPLOYED: Fri Sep 10 17:13:07 2021
NAMESPACE: karmada-system
STATUS: deployed
REVISION: 1
TEST SUITE: None
```
Pass test

File: hack/update-crdgen.sh [hunk 1]
```diff
@@ -11,5 +11,4 @@ source hack/util.sh
 
 echo "Generating with controller-gen"
 util::install_tools ${CONTROLLER_GEN_PKG} ${CONTROLLER_GEN_VER} >/dev/null 2>&1
-controller-gen crd paths=./pkg/apis/... output:crd:dir=./artifacts/crds/bases
-controller-gen crd paths=./pkg/apis/... output:crd:dir=./charts/_crds
+controller-gen crd paths=./pkg/apis/... output:crd:dir=./charts/_crds/bases
```
[the remark this answers is not in the document]
Comment (RainbowMango) [reply hack/update-crdgen.sh:14]: Sorry, I didn't get your point. The `chart` still `in the current project`, what `confusion` would be?

[the remark this answers is not in the document]
Comment (algebra2k) [reply hack/update-crdgen.sh:16]: > Sorry, I didn't get your point. The `chart` still `in the current project`, what `confusion` would be?

It's great for users, I mean for developers, crds generated directly into the chats directory may be a little not accustomed ,  so I feel like can add a comment to description.

**Comment by algebra2k** [comment]
> Installation scripts:
> 
> ```
> root@DESKTOP-TQOTF9Q:/workspace/projects/karmada# hack/remote-up-karmada.sh /root/.kube/config huawei-dev1
> Generating a RSA private key
> ....................+++++
> ...................+++++
> writing new private key to '/root/.karmada/server-ca.key'
> -----
> 2021/09/10 17:00:37 [INFO] generate received request
> 2021/09/10 17:00:37 [INFO] received CSR
> 2021/09/10 17:00:37 [INFO] generating key: rsa-2048
> 2021/09/10 17:00:37 [INFO] encoded CSR
> 2021/09/10 17:00:37 [INFO] signed certificate with serial number 443313073895049221857854521357943746403274820507
> Warning: resource namespaces/karmada-system is missing the kubectl.kubernetes.io/last-applied-configuration annotation which is required by kubectl apply. kubectl apply should only be used on resources created declaratively by either kubectl create --save-config or kubectl apply. The missing annotation will be patched automatically.
> namespace/karmada-system configured
> serviceaccount/karmada-controller-manager created
> serviceaccount/karmada-scheduler created
> serviceaccount/karmada-webhook created
> clusterrole.rbac.authorization.k8s.io/karmada-controller-manager created
> clusterrolebinding.rbac.authorization.k8s.io/karmada-controller-manager created
> secret/karmada-cert-secret created
> secret/kubeconfig created
> secret/webhook-cert created
> statefulset.apps/etcd created
> service/etcd-client created
> service/etcd created
> wait the etcd ready...
> ...
> ```
> 
> Helm:
> 
> ```
> root@DESKTOP-TQOTF9Q:/workspace/projects/karmada# helm install karmada -n karmada-system --create-namespace ./charts/
> NAME: karmada
> LAST DEPLOYED: Fri Sep 10 17:13:07 2021
> NAMESPACE: karmada-system
> STATUS: deployed
> REVISION: 1
> TEST SUITE: None
> ```
> 
> Pass test


/LGTM

**Comment by XiShanYongYe-Chang** [comment]
Hi @jrkeen , you might want to squash you commits :).

**Comment by jrkeen** [comment]
> Hi @jrkeen , you might want to squash you commits :).

Get

**Review by algebra2k — APPROVED** [review]

**Review by RainbowMango — COMMENTED** [review]
/approve

**Pull request merged**